src/pmdas/gfs2/pmda.c

```diff
@@ -267,6 +266,19 @@ pmdaMetric metrictable[] = {
         PMDA_PMID(CLUSTER_GLSTATS, GLSTATS_JOURNAL),
         PM_TYPE_U64, GFS_FS_INDOM, PM_SEM_INSTANT,
         PMDA_PMUNITS(0,0,1,0,0,PM_COUNT_ONE) }, },
+    /* GLOCKFD*/
+    { .m_desc = {
+        PMDA_PMID(CLUSTER_GLOCKFD, GLOCKFD_TOTAL),
+        PM_TYPE_U64, GFS_FS_INDOM, PM_SEM_INSTANT,
+        PMDA_PMUNITS(0,0,1,0,0,PM_COUNT_ONE) }, },
+    { .m_desc = {
+        PMDA_PMID(CLUSTER_HOLDER_STATS, GLOCKFD_PROCESS),
+        PM_TYPE_U64, GFS_HOLDER_INDOM, PM_SEM_INSTANT,
```

Comment:
PM_TYPE_32 here (and below).  And I think the units should be all zeros, not with 1/PM_COUNT_1 anywhere (again below as well).